pkgs/top-level/python-packages.nix

```diff
@@ -6511,9 +6511,7 @@ self: super: with self; {
 
   k-diffusion = callPackage ../development/python-modules/k-diffusion { };
 
-  k5test = callPackage ../development/python-modules/k5test {
-    inherit (pkgs) krb5 findutils;
```

Comment:
This is exactly how it should be done.

Reply:
If you prefer we can use `krb5-c = pkgs.krb5`.

Reply:
`krb-c` seems like a decent compromise, so at least the build will fail if you don't override the input instead of trying to use the wrong thing.

Reply:
Done